### PR TITLE
Final details

### PR DESCRIPTION
**Pull request by sz16900**
Cleaned the files with linters and validators. Update the README file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,657 @@
...
+                    </path>
+                </svg>
+            </div>
```
Comment (pelzolga123) [inline index.html]: NYT logo should be a link.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project needs some changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: great!
But still it needs some changes: 

* [x] Stickler is present but not set up.

* [x] The NY Times logo should be a link.

* [x] Main content should contain centered text and images
Right now it looks not centered:
image
image

* [x] Bottom part should be aligned like on original website:
image

Original:
image

* [x] Bottom part should contain links to other articles with image previews on the leftmost side.

* [x] Footer should have a fixed position (it sticks to the bottom part of the screen):
image

### Please kindly Fix the above issue and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by MauricioRobayo — APPROVED** [review]
🎅 All issues are solved and everything looks as it should be 😄 .

In fact, the page looks great! Awesome work, keep up the energy 🎆 .

**Status**: Approve.

**Pull request merged**